bundled/tool/lsp_server.py

```diff
@@ -544,6 +550,11 @@ def _run_tool_on_document(
     # deep copy here to prevent accidentally updating global settings.
     settings = copy.deepcopy(_get_settings_by_document(document))
 
+    if not settings["enabled"]:
+        log_warning(f"Skipping file [Linting Disabled]: {document.path}")
```

Comment:
is there reason why we have two separate log_warning? Is it to distinguish between f string and `

Reply:
Long lines are hard to read in the logs.